### PR TITLE
[WIP] Wheels: Use ocl-icd and bundle license

### PR DESCRIPTION
**Pull request by inducer**
Reverts inducer/pyopencl#271

### CODE REVIEW

File: travis/build-wheels.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+make -j4
+make install
+
+cd /io
+
+# Compile wheels
+for PYBIN in /opt/python/*/bin; do
+    "${PYBIN}/pip" install numpy pybind11 mako
+    "${PYBIN}/pip" wheel /io/ -w wheelhouse/
+done
```
Comment (isuruf) [inline travis/build-wheels.sh:44]: Can you change the above 4 lines to,

```bash
for PYBIN in /opt/python/*/bin; do
    if [[ "${PYBIN}" == *cp36* ]]; then
        NUMPY_VERSION="1.11.3"
    elif [[ "${PYBIN}" == *cp37* ]]; then
        NUMPY_VERSION="1.14.5"
    elif [[ "${PYBIN}" == *cp35* ]]; then
        NUMPY_VERSION="1.9.3"
    else
        NUMPY_VERSION="1.8.2"
    fi
    # Build with the oldest numpy available to be compatible with newer ones
    "${PYBIN}/pip" install "numpy==${NUMPY_VERSION}" pybind11 mako
    "${PYBIN}/pip" wheel /io/ -w wheelhouse/
done
```

Comment (isuruf) [reply travis/build-wheels.sh:44]: Or I can send a newer PR. This is the only thing I needed to change in the last PR.

Comment (inducer) [reply travis/build-wheels.sh:44]: Like this?

Comment (isuruf) [reply travis/build-wheels.sh:44]: yes, thanks. let me run this on a docker container and confirm.

Comment (isuruf) [reply travis/build-wheels.sh:44]: Needed a few other changes and pushed to https://github.com/inducer/pyopencl/pull/273

**Comment by inducer** [comment]
Closing in favor of #273.

**Pull request closed without merging**